packages/utils/src/SDKErrors.ts

```diff
@@ -174,9 +174,21 @@ export class IdentityMismatchError extends SDKError {
   }
 }
 
-export class WsAddressNotSetError extends SDKError {
-  constructor() {
-    super('Node address to connect to not configured!')
+export class FullnodeConnectionNotSetError extends SDKError {
```

Comment:
Shall we capitalize the word?
```suggestion
export class FullNodeConnectionNotSetError extends SDKError {
```

Reply:
could do, I wasn't sure if full node is one word or not